.gitignore

```diff
@@ -1,3 +1,4 @@
 target
 Cargo.lock
-/.vscode
\ No newline at end of file
+/.vscode
+nova_vm/src/heap/heap_bits.rs
```

Comment:
```suggestion
```

Reply:
Wooooooot? I did not put this in here 💀 I swear...

Reply:
:suspect: